pootle/apps/pootle_app/views/admin/dashboard.py

```diff
@@ -42,231 +41,6 @@
-        state = 'error'
-
-    required.append({
-        'dependency': 'redis',
-        'state': state,
-        'text': text,
-    })
-
-    status_workers, connection_settings['num'] = depcheck.test_rq_workers_running()
-    if status_workers:
-            text = ungettext(
-                    '%(num)d RQ worker running.',
-                    '%(num)d RQ workers running.',
-                    connection_settings['num'], connection_settings)
-            state = 'tick'
-    else:
-        text = _('No RQ workers are running.')
-        state = 'error'
-    required.append({
-        'dependency': 'rq',
-        'state': state,
-        'text': text,
-    })
-
-    if depcheck.test_cache():
-        if not depcheck.test_cache_server_connection():
-            # Server configured but connection failing
-            required.append({
-                'dependency': 'cache',
-                'state': 'error',
-                'text': _("Pootle is configured to use Redis as a caching "
-                          "backend, but can't connect to the cache.")
-            })
-        else:
-            required.append({
-                'dependency': 'cache',
-                'state': 'tick',
-                'text': _("Caching configured and running.")
-            })
-    else:
-        required.append({
-            'dependency': 'cache',
-            'state': 'error',
-            'text': _("Redis is required as the caching backend.")
-        })
-
-
-    return required
-
-
-def optional_depcheck():
-    optional = []
-
-    if not depcheck.test_iso_codes():
-        optional.append({
-            'dependency': 'iso-codes',
-            'text': _("Can't find the ISO codes package. Pootle uses ISO codes"
-                      " to translate language names.")
-        })
-    if not depcheck.test_levenshtein():
-        optional.append({
-            'dependency': 'levenshtein',
-            'text': _("Can't find python-levenshtein package. Updating against"
-                      " templates is faster with python-levenshtein.")
-        })
-
-    filter_name, filter_args = get_markup_filter()
-    if filter_name is None:
-        text = None
-        if filter_args == 'missing':
-            text = _("MARKUP_FILTER is missing. Falling back to HTML.")
-        elif filter_args == 'misconfigured':
-            text = _("MARKUP_FILTER is misconfigured. Falling back to HTML.")
-        elif filter_args == 'uninstalled':
-            text = _("Can't find the package which provides '%s' markup "
-                     "support. Falling back to HTML.",
-                     settings.MARKUP_FILTER[0])
-        elif filter_args == 'invalid':
-            text = _("Invalid value '%s' in MARKUP_FILTER. Falling back to "
-                     "HTML.", settings.MARKUP_FILTER[0])
-
-        if text is not None:
-            optional.append({
-                'dependency': filter_args + '-markup',
-                'text': text
-            })
```

Comment:
If the current markup depchecks are not suitable to be ported as is, then they should be adjusted. I do agree that maybe not all these checks are worth porting, but we should at least ensure that if `MARKUP_FILTER` tells to use reStructuredText (or Textile or Markdown) the required package is available.